### ISSUE TITLE
Big Dipper Latency

### ISSUE THREAD

**Issue by justyntemme** [Pylons-tech/pylons#620]
## Summary of Bug
Currently, big dipper transactions can take up to two minutes to load (sometimes closer to 10 min) https://wallet.pylons.tech/transactions

This isn't always the case and is intermittent. However, I feel it is related to the other latency issue where the block explorer cannot keep up with the latest block height. As a temporary solution, we have installed mongodb on the same server big dipper runs on. This seems to give us the required speed to keep up with the latest blocks, but is not the best solution, and requires we ignore kubernetes and run on a vm instance giving up all of our great pipelines and automation.

TLDR; Big dipper can't always keep up to the latest block heigh unless mongodb is installed alongside BD on the same vm instance. When it is able to keep up, transactions can take up to 2-10 minutes to load.

@faddat has suggested this is because we are hitting a validator nodes gRPC endpoint, causing delays in the query processing on the validator.

**Comment by faddat** [comment]
We have provided new endpoints, did you get them?
